cobbler.spec

```diff
@@ -40,7 +40,7 @@ Summary: Boot server configurator
 Name: cobbler
 License: GPLv2+
 Version: 2.9.0
-Release: 1%{?dist}
+Release: 8%{?dist}
```

Comment:
Reset this back to `1%{?dist}`. That should stay put unless there's a really good reason not to.

Reply:
Yes, but the first part is a valid fix we'd like to have.